defaults/main.yml

```diff
@@ -16,6 +16,46 @@ nginx_http_params:
+              ~*(ContextAd\ Bot|Digincore\ bot|Disqus|Feedly|Fetch\/|Fever|Flamingo_SearchEngine|FlipboardProxy|g2reader-bot|G2\ Web\ Services|imrbot|K7MLWCBot|Kemvibot|Landau-Media-Spider|linkapediabot|vkShare|Siteimprove.com|BLEXBot\/|DareBoost|ZuperlistBot\/|Miniflux\/|Feedspot|Diffbot\/|SEOkicks|tracemyfile)  1;
+              ~*(Nimbostratus-Bot|zgrab|PR-CY.RU|AdsTxtCrawler|Datafeedwatch|TangibleeBot|google-xrawler|axios|Amazon\ CloudFront|Pulsepoint|CloudFlare-AlwaysOnline|Google-Structured-Data-Testing-Tool|WordupInfoSearch|WebDataStats|HttpUrlConnection|Seekport\ Crawler|ZoomBot|VelenPublicWebCrawler|MoodleBot)  1;
+              ~*(jpg-newsbot|outbrain|W3C_Validator|Validator\.nu|W3C-checklink|W3C-mobileOK|W3C_I18n-Checker|FeedValidator|W3C_CSS_Validator|W3C_Unicorn|Google-PhysicalWeb|Blackboard|ICBot\/|BazQux|Twingly|Rivva|Experibot|awesomecrawler|Dataprovider.com|GroupHigh\/|theoldreader.com|AnyEvent|Uptimebot\.org|Nmap\ Scripting\ Engine)  1;
+              ~*(2ip.ru|Clickagy|Caliperbot|MBCrawler|online-webceo-bot|B2B\ Bot|AddSearchBot|Google\ Favicon|HubSpot|Chrome-Lighthouse|HeadlessChrome|CheckMarkNetwork\/|www\.uptime\.com|Streamline3Bot\/|serpstatbot\/|MixnodeCache\/|SimpleScraper|RSSingBot|Jooblebot|fedoraplanet|Friendica|NextCloud|Tiny\Tiny\ RSS)  1;
+              ~*(RegionStuttgartBot|Bytespider|Datanyze|Google-Site-Verification|TrendsmapResolver|tweetedtimes|NTENTbot|Gwene|SimplePie|SearchAtlas|Superfeedr|feedbot|UT-Dorkbot|Amazonbot|SerendeputyBot|Eyeotabot|officestorebot|Neticle\ Crawler|SurdotlyBot|LinkisBot|AwarioSmartBot|AwarioRssBot|RyteBot|FreeWebMonitoring\ SiteChecker)  1;
+              ~*(AspiegelBot|NAVER\ Blog\ Rssbot|zenback\ bot|SentiBot|Domains\ Project\/|Pandalytics|VKRobot|bidswitchbot|tigerbot|NIXStatsbot|Atom\ Feed\ Robot|Curebot|PagePeeker\/|Vigil\/|rssbot\/|startmebot\/|JobboerseBot|seewithkids|NINJA\ bot|Cutbot|BublupBot|BrandONbot|RidderBot|YandexMetrika)  1;
+              ~*(YandexTurbo|YandexImageResizer|YandexVideoParser|Taboolabot|Dubbotbot|FindITAnswersbot|infoobot|Refindbot|BlogTraffic\/\d\.\d+\ Feed-Fetcher|SeobilityBot|Cincraw|Dragonbot|VoluumDSP-content-bot|FreshRSS|BitBot|PetalBot|SemrushBot) 1;
+            }
+            map $isbot_ua $limit_bot {
+              0       "";
+              # Use $http_x_forwarded_for instead of $binary_remote_addr when using reverse proxy
+              1       $binary_remote_addr;
+            }
+            limit_req_zone $limit_bot zone=bots:10m rate=1r/m
+  # Use $http_x_forwarded_for instead of $binary_remote_addr when using reverse proxy
+  - limit_req_zone $binary_remote_addr zone=one:50m rate=50r/s
+  - limit_req zone=bots burst=2 nodelay
+  - limit_req zone=one burst=15 nodelay
```

Comment:
A burst limit of 15 requests seems too low to me for a request rate limit of 50r/s.  I don't totally understand how burst limits work from the Nginx docs (http://nginx.org/en/docs/http/ngx_http_limit_req_module.html) but I think the burst limit should be greater or equal to the rate limit.